stack/pico_device.c

```diff
@@ -85,11 +85,12 @@ static int pico_6lowpan_store_info(struct pico_device *dev, const uint8_t *mac)
 #ifdef PICO_SUPPORT_IPV6
 static void device_init_ipv6_final(struct pico_device *dev, struct pico_ip6 *linklocal)
 {
+    IGNORE_PARAMETER(linklocal);
+
     dev->hostvars.basetime = PICO_ND_REACHABLE_TIME;
     /* RFC 4861 $6.3.2 value between 0.5 and 1.5 times basetime */
-    dev->hostvars.reachabletime = ((5 + (pico_rand() % 10)) * PICO_ND_REACHABLE_TIME) / 10;
+    dev->hostvars.reachabletime = ((5 + (pico_rand() % 10)) * dev->hostvars.basetime) / 10;
```

Comment:
I'd be curious if this change gives a different compiler output. If we have a smart compiler, it should not...